com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyTextDocumentService.java

```diff
@@ -194,6 +191,7 @@ public void didChange(DidChangeTextDocumentParams params) {
   public void didClose(DidCloseTextDocumentParams params) {
     String uri = params.getTextDocument().getUri();
     log.info("Document closing invoked");
```

Comment:
Maybe, we may log also the URI here

Reply:
It wasn't part of my activity, but definitely I could do it